tests/models/test_DurableOrchestrationClient.py

```diff
@@ -540,3 +543,51 @@ async def test_start_new_orchestrator_internal_exception(binding_string):
     with pytest.raises(Exception) as ex:
         await client.start_new(TEST_ORCHESTRATOR)
     ex.match(status_str)
+
+@pytest.mark.asyncio
+async def test_rewind_works_under_200_and_200_http_codes(binding_string):
+    """Tests that the rewind API works as expected under 'successful' http codes: 200, 202"""
+    client = DurableOrchestrationClient(binding_string)
+    for code in [200, 202]:
+        mock_request = MockRequest(
+            expected_url=f"{RPC_BASE_URL}instances/{INSTANCE_ID}/rewind?reason={REASON}",
+            response=[202, ""])
```

Comment:
nit: should this line be `response=[code,""])`

Reply:
Ooh, silly me. That's a refactor gone wrong, thank you!

Reply:
Done!